src/movies.js

```diff
@@ -1,16 +1,89 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+function getAllDirectors(movies){
+    let answer1 = movies.map((movie) => {
+        return movie.director;
+    })
+return answer1;
+};
+
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
 
+function howManyMovies(movies){
+    let drama;
+    let spiel = movies.filter((movie) => {
+    for (let i=0; i < movie.genre.length; i++){
```

Comment:
The loop works well here, but you could use the method .includes to check the "Drama" inside the genre array. Looks more clean and you avoid running the loop.